controllers/sms-gateway.js

```diff
@@ -73,9 +80,11 @@ function getOutgoing(callback) {
         content: message.message,
       };
     });
-    markMessagesForwarded(messages, function() {
+    // TODO add this back in once this function also queries for
+    // messages in state: 'forwarded-to-gateway'
+    // markMessagesForwarded(messages, function() {
```

Comment:
Can you please remind me again why we just don't implement this?  The message API currently allows for setting the state to any arbitrary string and querying for it.   The docs don't state that, but I just tried it.  https://github.com/medic/medic-api#messages